reactive/docker.py

```diff
@@ -49,6 +49,10 @@ def install():
     charm_path = os.getenv('CHARM_DIR', '')
     install_script_path = os.path.join(charm_path, 'scripts/install_docker.sh')
     check_call([install_script_path])
+
+    opts = DockerOpts()
+    render('docker.defaults', '/etc/default/docker', {'opts': opts.to_s()})
```

Comment:
Wouldn't we want to do the rendering of options before we short circuit on line 45?

I didn't write this code (so not familiar). Does the install script start the docker daemon? If so shouldn't we configure the options before the daemon starts?